tests/manager/manager-ctor-ssl-003.phpt

```diff
@@ -0,0 +1,23 @@
+--TEST--
+MongoDB\Driver\Manager::__construct(): Specifying a driver option implicitly enables TLS
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_not_libmongoc_ssl(['OpenSSL', 'LibreSSL']); ?>
+<?php skip_if_ssl(); ?>
+--FILE--
+<?php
+
+require_once __DIR__ . "/../utils/basic.inc";
+
+$manager = new MongoDB\Driver\Manager(URI, [], ['ca_dir' => 'foo']);
```

Comment:
I think this test should also assert that we _can_ connect and ping the URI as-is. Otherwise, this test may pass by virtue of the server being down.

You can also add a note that this test would fail if the the server was configured with either `allowSSL` or `preferSSL` for [`net.ssl.mode`](https://docs.mongodb.com/manual/reference/configuration-options/#net.ssl.mode).